xds/internal/resolver/xds_resolver_test.go

```diff
@@ -57,11 +57,10 @@ var (
 )
 
 func TestRegister(t *testing.T) {
-	for _, s := range []string{"xds", "xds-experimental"} {
-		b := resolver.Get(s)
-		if b == nil {
-			t.Errorf("scheme %v is not registered", s)
-		}
+	s := xdsScheme
```

Comment:
Nit: Do we really need `s`? Can't we directly use `xdsScheme`?

Reply:
Done